packages/vkui/src/components/SubnavigationBar/SubnavigationBar.tsx

```diff
@@ -48,14 +48,16 @@ export const SubnavigationBar = ({
     };
   }
 
+  const actualChildren = React.useMemo(() => React.Children.toArray(children), [children]);
+
   return (
     <RootComponent
       baseClassName={mode === 'fixed' && styles['SubnavigationBar--mode-fixed']}
       {...restProps}
     >
       <ScrollWrapper className={styles['SubnavigationBar__in']} {...scrollWrapperProps}>
         <ul className={styles['SubnavigationBar__scrollIn']}>
-          {React.Children.map(children, (child, idx) => (
+          {actualChildren.map((child, idx) => (
             <li key={idx} className={styles['SubnavigationBar__item']}>
               {child}
             </li>
```

Comment:
```suggestion
          {React.Children.map((children, (child, idx) => hasReactNode(child) ? (
            <li key={idx} className={styles['SubnavigationBar__item']}>
              {child}
            </li>
          ) : null}
```

мб так?

ЗЫ: мемоизация `children` бессмысленна, т.к. в случае с `SubnavigationBar` постоянно новый массив будет приходить :(